packages/react-dom/src/client/ReactDOMFiberSelect.js

```diff
@@ -182,6 +182,8 @@ export function initWrapperState(element: Element, props: Object) {
 
 export function postMountWrapper(element: Element, props: Object) {
   const node = ((element: any): SelectWithWrapperState);
+  //Set selected index to -1 before doing anything else to prevent first option from being selected by default
+  node.selectedIndex = -1;
   node.multiple = !!props.multiple;
```

Comment:
Possibly chasing ghosts, but I'm worried about the side-effects of always assigning selectedIndex to -1. Do you think it makes sense to only apply this for the multiple input case?

```javascript
const multiple =  !!props.multiple;

if (multiple) {
  node.selectedIndex = -1
  node.multiple = true
}
```

Reply:
Good suggestion, since this only occurs in the event of multiple this makes sense.